packages/cketh/candid/minter.certified.idl.js

```diff
@@ -126,6 +134,7 @@ export const idlFactory = ({ IDL }) => {
         'address' : IDL.Text,
         'ckerc20_token_symbol' : IDL.Text,
       }),
+      'QuarantinedDeposit' : IDL.Record({ 'event_source' : EventSource }),
```

Comment:
I believe adding a new variant to an existing type is a breaking change. I'm checking with Greg if this is OK.

Reply:
Greg answered that this is only used in a debug endpoint. So it's fine.

Does this PR make https://github.com/dfinity/ic-js/pull/613 obsolete?

Reply:
> Does this PR make https://github.com/dfinity/ic-js/pull/613 obsolete?

No. It contains other changes for the ledger notably.

Thanks for checking with Greg.